scale/storage/models.py

```diff
@@ -412,6 +426,13 @@ class ScaleFile(models.Model):
     job_exe = models.ForeignKey('job.JobExecution', blank=True, null=True, on_delete=models.PROTECT)
     job = models.ForeignKey('job.Job', blank=True, null=True, on_delete=models.PROTECT)
     job_type = models.ForeignKey('job.JobType', blank=True, null=True, on_delete=models.PROTECT)
+    job_output = models.CharField(null=True, blank=True, max_length=250)
+    recipe = models.ForeignKey('recipe.Recipe', null=True, blank=True, db_index=True, on_delete=models.PROTECT)
```

Comment:
db_index=True is not needed for ForeignKey fields, it is automatic.
Applies to recipe, recipe_type, and batch.